documentation/api/reference/made_with_ov.md

```diff
@@ -0,0 +1,76 @@
+# Made with Openverse
+
+The Openverse API provides access to the world's largest library of
+openly-licensed creative works, containing over 700 million images and audio
+tracks. All Openverse content is under a
+[Creative Commons license](https://creativecommons.org/licenses/) or is in the
+public domain.
```

Comment:
This probably needs a caveat, either in this document _or_ a link to a general document stating that Openverse does not make any claims about the licence of works and licences should always be verified with providers/creators for absolute certainty.

Reply:
In that case the Openverse homepage will need a caveat too, because it makes the same remark.

<img width="795" alt="Screenshot 2023-08-28 at 4 26 21 PM" src="https://github.com/WordPress/openverse/assets/16580576/dfb5d8ce-0c2c-4be4-ac88-850d5e00801e">

Reply:
Added in 9f702a4.

Reply:
> In that case the Openverse homepage will need a caveat too, because it makes the same remark.

Yes, absolutely, it is a wider problem.